projects/packages/search/src/instant-search/components/overlay.scss

```diff
@@ -44,7 +44,34 @@ $overlay-horizontal-padding-lg: 3em;
 		padding: $overlay-vertical-padding-lg $overlay-horizontal-padding-lg;
 	}
 
-	h1, h2, h3, h4, h5, h6 {
+	// TODO: look into a more sustainable approach to reduce the number of !important being used to override the button styles
+	button {
+		background-color: inherit !important;
+		color: $color-text-lighter !important;
```

Comment:
```suggestion
		color: inherit !important;
```

Shall we just go with `inherit` here? `$color-text-lighter` seems to be really dark in dark mode.

<img width="493" alt="Screen Shot 2022-08-15 at 15 53 02" src="https://user-images.githubusercontent.com/17325/184574378-436c1a18-63f8-4671-865f-49e7e6231cb3.png">